ext/search_kit/ang/crmSearchTasks/crmSearchInput/crmSearchInput.component.js

```diff
@@ -47,7 +47,7 @@
             formatted.forEach((v, i) => formatted[i] = formatDataType(v));
             return formatted;
           }
-          if (ctrl.field.data_type === 'Integer' || ctrl.field.data_type === 'Float') {
+          if (['Integer', 'Float'].includes(ctrl.field?.data_type)) {
```

Comment:
My ide says this only works in ES11. I'm not sure if that's ok but to be safe we could do this:
```suggestion
          if (['Integer', 'Float'].includes(ctrl.field ? ctrl.field.data_type : null)) {
```